docs/DeveloperGuide.md

```diff
@@ -484,25 +489,83 @@ Extensions:
 
 **Use case: UC0X - Create a session**
 
-**Preconditions: Student profile linked to session has been created.**
+MSS:
+
+1. User enters the `add_session` command, together with the session details.
+2. TutorBuddy verifies that the student exists, and the inputs are valid.
+3. TutorBuddy creates the session.
+4. TutorBuddy displays a success message.
+
+   Use case ends.
+
+**Extensions:**
```

Comment:
actually do we need to create an extension for students don't exist?

Reply:
I tink previously step 2 is split into 2, it checks for student info first and then checks for valid input. But then we feel that the "checking" part can be consolidated into 1 single step and just classify it generically as "detects an error", cos when students don't exist, its technically an error as well. Hence this is what we have in the end. Or do you feel that we should still spilt them up again?

Reply:
oh yes, actually invalid student can be considered input as well. ok then! let's remain.